src/single-line-diagram-viewer.ts

```diff
@@ -233,8 +233,8 @@ export class SingleLineDiagramViewer {
             //so that panzoom's margins still work correctly.
             //I am not sure the offsetX and offsetY thing is correct. It seems
             //to help. When someone finds a big problem, then we can fix it.
-            const newLvlX = svgWidth / maxWidth / 2;
-            const newLvlY = svgHeight / maxHeight / 2;
+            const newLvlX = svgWidth / (maxWidth + 200);
```

Comment:
The formula works fine for some large SLD but not some others. 
By example, the rendering is incorrect for VLEJUP7 of a Recollement...

Reply:
Thanks for the feedback.
I made a couple of changes, then tried with these: VLEJUP7, VLEJU, VOUGL, VOID, ARGOE.
Now, to me, it seem indistinguishable from the rendering done by the current gridstudy-app.
Could you please take another look?